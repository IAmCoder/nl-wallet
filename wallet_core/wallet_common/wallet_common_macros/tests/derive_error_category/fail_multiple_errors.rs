#[derive(wallet_common_macros::ErrorCategory)]
enum Error {
    First,
    Second,
}

fn main() {
    let _ = Error::First;
}
