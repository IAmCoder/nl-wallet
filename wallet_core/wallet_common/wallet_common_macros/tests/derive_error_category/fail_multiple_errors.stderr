error: Enum variant is missing `category` attribute
 --> tests/derive_error_category/fail_multiple_errors.rs:3:5
  |
3 |     First,
  |     ^^^^^

error: Enum variant is missing `category` attribute
 --> tests/derive_error_category/fail_multiple_errors.rs:4:5
  |
4 |     Second,
  |     ^^^^^^
